packages/overlay/src/integrations/sentry/components/traces/TraceList.tsx

```diff
@@ -11,6 +12,21 @@ import { getDuration } from '../../utils/duration';
 import HiddenItemsButton from '../HiddenItemsButton';
 import TraceIcon from './TraceIcon';
 
+function TransactionName({ trace }: { trace: Trace }) {
+  const method = String(
+    trace.rootTransaction?.contexts?.trace.data?.method || trace.rootTransaction?.request?.method || '(GET)',
+  );
+  const name = trace.rootTransactionName.startsWith(method)
+    ? trace.rootTransactionName.slice(method.length + 1)
+    : trace.rootTransactionName;
+  return (
+    <div className="border-primary-300 bg-primary-900 divide-x-primary-300 text-smflex inline-flex divide-x whitespace-nowrap rounded-full border font-mono">
```

Comment:
the design of this is almost same as of Tag Component.
How about using it?

Reply:
Yup, I can make it generalized.

Reply:
For sure! I think my favorite is option 2. And I'm also okay with option 3 of you prefer that. Didn't like option 1 much 😅